frontend/apps/web/src/components/NumericInput.tsx

```diff
@@ -23,7 +24,7 @@ export const NumericInput: React.FC<NumericInputProps> = ({ value, onChange, ...
     const propagateChange = () => {
         const parsedValue = parseAbrechnungFloat(internalValue);
         if (!isNaN(parsedValue)) {
-            setInternalValue(String(parsedValue));
+            setInternalValue(isCurrency ? value.toFixed(2) : String(value));
```

Comment:
Small error here, it should be 

```ts
setInternalValue(isCurrency ? parsedValue.toFixed(2) : String(parsedValue));
```
instead.

Otherwise the old value will be temporarily shown after updating which will lead to a flicker in the display.

Reply:
Good catch, thanks!